GDJS/GDJS/IDE/ExporterHelper.cpp

```diff
@@ -768,6 +785,7 @@ void ExporterHelper::AddLibsInclude(bool pixiRenderers,
   InsertUnique(includesFiles, "inputmanager.js");
   InsertUnique(includesFiles, "jsonmanager.js");
   InsertUnique(includesFiles, "Model3DManager.js");
+  InsertUnique(includesFiles, "capturemanager.js");
```

Comment:
Would be good to add this only if there is a captureOptions passed. This will avoid yet another file and yet another request when loading (web) games